canary_endpoint/resources/databases.py

```diff
@@ -31,7 +31,7 @@ def check(self):
         result = super(Database, self).check()
         try:
             cursor = self.connection.cursor()
-            map(cursor.execute, self.statements)
+            [cursor.execute(sql) for sql in self.statements]
```

Comment:
How about a `for` loop instead? Executing as a by-product of a list comprehension looks like you are creating a variable that never gets assigned to anything (expected: `statements = [transform(sql) for sql in self.statements]`).